packages/plugin-assets/src/init.js

```diff
@@ -0,0 +1,78 @@
+const { TwilioServerlessApiClient } = require('@twilio-labs/serverless-api');
+const {
+  createService,
+} = require('@twilio-labs/serverless-api/dist/api/services');
+const {
+  createEnvironmentFromSuffix,
+  getEnvironment,
+} = require('@twilio-labs/serverless-api/dist/api/environments');
+const { TwilioCliError } = require('@twilio/cli-core').services.error;
+
+const DEFAULT_ASSET_SERVICE_NAME = 'CLI-Assets-Bucket';
+
+const createServiceAndEnvironment = async client => {
+  const serviceSid = await createService(DEFAULT_ASSET_SERVICE_NAME, client);
+  const environment = await createEnvironmentFromSuffix('', serviceSid, client);
+  return {
+    serviceSid,
+    environment,
+  };
+};
+
+const init = async ({
+  apiKey,
+  apiSecret,
+  accountSid,
+  pluginConfig,
+  logger,
+}) => {
+  logger.debug('Loading config');
+  const client = new TwilioServerlessApiClient({
+    username: apiKey,
+    password: apiSecret,
+  });
+  const config = await pluginConfig.getConfig();
+  if (
+    config[accountSid] &&
+    config[accountSid].serviceSid &&
+    config[accountSid].environmentSid
+  ) {
+    const { serviceSid, environmentSid } = config[accountSid];
+    logger.debug(
+      `Fetching environment with sid ${environmentSid} from service with sid ${serviceSid}`
+    );
+    try {
+      const environment = await getEnvironment(
+        environmentSid,
+        serviceSid,
+        client
+      );
+      return environment;
+    } catch (error) {
+      logger.debug(error.toString());
+      throw new TwilioCliError(
+        `Could not fetch asset service environment with config:
```

Comment:
It would be useful to provide some steps here as to why / how to fix it

Reply:
I've added a message to run in debug mode for more detail. Many of the errors throughout these scripts should not occur unless there is a network error or the user has edited the config file by hand and changed the SIDs.